pkg/urlmanager/urlmanager.go

```diff
@@ -10,14 +10,18 @@ import (
 )
 
 // GetDownloadURL returns file download url for given git provider
-func GetDownloadURL(gitprovider, repoSlug, commitID, fileName string) (string, error) {
+func GetDownloadURL(gitprovider, repoLink, repoSlug, commitID, fileName string) (string, error) {
```

Comment:
remove this function. It is not being used.

Reply:
sure

Reply:
Somebody has written a unit test for it and that has been merged in dev, should I revert my deletion or should I take a pull and delete that unit test as well @saurabh-prakash ?